cla_backend/apps/legalaid/models.py

```diff
@@ -428,30 +427,21 @@ def get_eligibility_state(self):
         logger.debug('CaseData %s' % json.dumps(case_data_dict, indent=4, sort_keys=True))
         logger.debug('CaseData is missing: %s' % json.dumps(case_data_dict_missing, indent=4, sort_keys=True))
         ec = EligibilityChecker(case_data)
-        eligibility_state = ec.is_eligible()
+        eligibility_state, is_gross_income_eligible, is_disposable_income_eligible, is_disposable_capital_eligible = ec.is_eligible_with_reasons()
```

Comment:
is it any neater if `is_eligible_with_reasons` returned (eligibilty_state, reasons)?

Reply:
seems fine